SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestNetReference/CubeWithReferences.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "CubeWithReferences.h"
+#include "Components/StaticMeshComponent.h"
+#include "Materials/Material.h"
+#include "Net/UnrealNetwork.h"
+
+ACubeWithReferences::ACubeWithReferences()
+{
+	bReplicates = true;
```

Comment:
This will need preprocessor guards for 4.23/4.24 and beyond, I think, to avoid deprecation warnings.
SetReplicated or something is the replacement?

Reply:
Oh, why is this the case, is it related to 4.25 changes? I will make sure to change that.

Reply:
Actually had a look, and I was wrong. Only bReplicateMovement needs to use SetReplicatingMovement in 4.24 and beyond, this one can be left in. But see if you have any warnings when compiling.

Reply:
Just checked, there are no warnings when compiling this.